centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -122,8 +123,8 @@ object Operations extends StrictLogging {
   // The project from the config or from the credentials. By default the project is read from the system environment.
   lazy val projectOption: Option[String] = confProjectOption orElse credentialsProjectOption
 
-  lazy val genomics: Genomics = {
-    val builder = new Genomics.Builder(
+  lazy val genomics: CloudLifeSciences = {
```

Comment:
Technically this is a `CloudLifeSciences` now and not a `Genomics`, even if neither of those terms really makes sense as an abstraction...